docs/UserGuide.adoc

```diff
@@ -528,16 +528,22 @@ There is no need to save manually.
 ===== Exporting contact data: `export`
 
 Exports all contact data to an external file. +
-Format: `export FILE_PATH`
+Command format: `export FILE_PATH` +
```

Comment:
Can we use the colors to denote (red)COMMAND_WORD, (blue)OPTION and (green)PARAMETERS?